uwsgi/run.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
+build_front_end() {
+   cd /app/ui
+   lineman build
+   cd -
+}
+
 if [ $1 == "public" ]; then
    exec uwsgi --ini /app/uwsgi/public.ini --python-autoreload 1
 elif [ $1 == "admin" ]; then
+   build_front_end
    exec uwsgi --ini /app/uwsgi/admin.ini --python-autoreload 1
 elif [ $1 == "admin-dev" ]; then
+   build_front_end
```

Comment:
Is there a reason you're building the frontend as part of the ENTRYPOINT instead of doing it during the build?